Electron/src/Core/components/ControlScheme.tsx

```diff
@@ -0,0 +1,291 @@
+// eslint-disable-next-line max-classes-per-file
+import React, { Component, ReactNode, useState } from "react"
+import CSS from "csstype"
+import { CONTROLLERINPUT } from "./ControllerInput"
+
+const container: CSS.Properties = {
+  display: "flex",
+  fontFamily: "arial",
+  borderTopWidth: "30px",
+  borderColor: "#990000",
+  borderBottomWidth: "2px",
+  borderStyle: "solid",
+  flexWrap: "wrap",
+  flexDirection: "column",
+}
+const label: CSS.Properties = {
+  marginTop: "-10px",
+  position: "relative",
+  top: "24px",
+  left: "3px",
+  fontFamily: "arial",
+  fontSize: "16px",
+  zIndex: 2,
+  color: "white",
+}
+const readoutDisplay: CSS.Properties = {
+  fontSize: "16px",
+  fontFamily: "arial",
+  padding: "6px 3px 3px 4px",
+  marginRight: "2px",
+  flex: 1,
+}
+const row: CSS.Properties = {
+  display: "flex",
+  flexDirection: "row",
+  margin: "2px",
+}
+
+// eslint-disable-next-line import/no-mutable-exports
+export let inputs: any = {}
```

Comment:
name this something more descriptive like controllerInputs

Reply:
Changed it to controllerInputs in ControlScheme.tsx